reports/post_diagnostics.Rmd

```diff
@@ -246,6 +245,28 @@ kable(measure_decline_df, format.args = list(big.mark = ",")) %>%
 
 ---
 
+# Comparison to CPP 
+``` {r, echo = F}
+cpp_df <- read_cpp_data()
+
+new_state_df %>% 
+    left_join(cpp_df %>% 
+                  select(-Date) %>% 
```

Comment:
What do you think of holding onto the `Date` last updated from CPP? My thought is that our scraping schedules won't always perfectly align, and keeping both dates could be a helpful reminder about why the difference may not equal zero

Reply:
Good call – done in the latest commit!